proposals/stage-0/TRAVEL_FUND/README.md

```diff
@@ -0,0 +1,56 @@
+# OpenJS Travel Fund
+>  Stage 0
+
+Tracked by https://github.com/openjs-foundation/cross-project-council/issues/172
+
+## Champion
+
+Jonah Stiennon @jonahss
+
+## Description
+
+The Node.js organizations that existed prior to the formation of the OpenJS foundation oversaw a monetary fund for the purpose of member and affiliate travel to, and lodging at, events related to the foundation's mission. This proposal proposes moving the management responsibilities of this fund to the Cross Project Council (CPC), and making the funds available to all members of this broader organization. Since all projects have voting representation on the CPC, the beneficiaries of the fund will jointly manage it, and the previous owners of the fund will still vote on its use.
+
+We shall review the process in this proposal, but the intent is to leave the mechanics of the fund unchanged, simply moving from the purview of the Node.js project into the domain of the OpenJS Foundation, overseen by the CPC. This operation can be viewed as a refactor which moves the travel fund up one level in the organizational tree.
+
+The current fund is used mostly for attending collaborator summits, and we do not expect this to change.
+
+## Difference from the current process
+
+The existing process requires approval from two members of the Node.js TSC and two members of the Node.js Community Committee.
```

Comment:
It would be good to have links to these groups.

Reply:
added.